File: pytools/data/data.code.java.client/ReqRoleRandName.java
package protocol;


import network.Packet;
import network.PacketUtil;

import java.util.List;
import java.util.ArrayList;


public class ReqRoleRandName
{


	public byte[] encode()
	{
		Packet packet = new Packet();
		return packet.encode((short)Msg.P_REQ_ROLE_RAND_NAME);
	}



}

File: pytools/data/data.code.golang.client/req_role_rand_name.go
package proto

import (
	"packet"
)

type ReqRoleRandName struct {
}

func (this *ReqRoleRandName) Encode() []byte {
	pack := packet.NewWriteBuff(64)


	return pack.Encode(P_REQ_ROLE_RAND_NAME)
}
